modules/assim.batch/R/pda.emulator.R

```diff
@@ -55,10 +64,10 @@ pda.emulator <- function(settings, params.id = NULL, param.names = NULL, prior.i
     con <- NULL
   }
 
-  bety <- PEcAn.visualization::betyConnect("~/pecan/web/config.php")
+  bety <- betyConnect("~/pecan/web/config.php")
```

Comment:
Asking everyone to use this syntax everywhere is clunky and inefficient.  I'd recommend that we modify our existing db connection functions to work with the new dplyr connection. For example, betyConnect could  be overloaded such that it does it's current behavior is passed a string (path) or it does the above if passed a list (settings$database$bety). Or we could overload our existing db.open to accept a path to a config.php. Either way, we need to get down to one connection function and no manual database opens